### PR TITLE
Lesson 6 task 6 - Conversion to Uppercase

### PR DESCRIPTION
**Pull request by jean-farinea**
1. Created class to convert text to uppercase and write to a new file.

### CODE REVIEW

File: src/main/java/lesson6/task6/ConvertToUppercase.java [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+    public static void convertToUpperCase() throws IOException {
+        String filePath = "src/main/resources/data.txt";
+        String content = FileUtils.readFromFile(filePath);
+        String newContentFormatted = content.toUpperCase();
+        FileUtils.writeToFile("src/main/resources/output.txt", newContentFormatted);
+    }
+
+    public static void main(String[] args) throws IOException {
+        convertToUpperCase();
```
Comment (strzeleckik) [inline src/main/java/lesson6/task6/ConvertToUppercase.java]: test should be in unit tests

File: src/main/java/lesson6/task6/ConvertToUppercase.java [hunk 2]
```diff
@@ -0,0 +1,19 @@
+package lesson6.task6;
+
+import lesson6.task0.FileUtils;
+
+import java.io.IOException;
+
+public class ConvertToUppercase {
+
+    public static void convertToUpperCase() throws IOException {
+        String filePath = "src/main/resources/data.txt";
```
Comment (strzeleckik) [inline src/main/java/lesson6/task6/ConvertToUppercase.java]: to param

File: src/main/java/lesson6/task6/ConvertToUppercase.java [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+        String content = FileUtils.readFromFile(filePath);
+        String newContentFormatted = content.toUpperCase();
+        FileUtils.writeToFile("src/main/resources/output.txt", newContentFormatted);
```
Comment (strzeleckik) [inline src/main/java/lesson6/task6/ConvertToUppercase.java]: to param

**Comment by jean-farinea** [comment]
Adjusted. Please review.

**Review by strzeleckik — APPROVED** [review]
ok

**Pull request merged**